neqo-common/src/codec.rs

```diff
@@ -363,6 +363,12 @@ impl Debug for Encoder {
     }
 }
 
+impl AsRef<[u8]> for Encoder {
```

Comment:
do we need this for this PR?

Reply:
No, I mixed the streams, but it's not a big deal if it is here.